OpenSim/Simulation/SimbodyEngine/Joint.h

```diff
@@ -250,8 +254,19 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Joint, ModelComponent);
     calcEquivalentSpatialForce(const SimTK::State &s, 
                                const SimTK::Vector &mobilityForces) const;
 
+    // Reaction forces (direct calls into Simbody methods)
+    /** Reaction forces on Child Body at Origin in Ground frame @see SimTK::MobilizedBody method. */
+    SimTK::SpatialVec
+        calcReactionOnChildAtOriginInGround(const SimTK::State &s) const {
+        return getChildFrame().getMobilizedBody().findMobilizerReactionOnBodyAtOriginInGround(s);
+    }
+    /** Reaction forces on Parent Body at Origin in Ground frame @see SimTK::MobilizedBody method. */
+    SimTK::SpatialVec
+        calcReactionOnParentAtOriginInGround(const SimTK::State &s) const {
+        return getChildFrame().getMobilizedBody().findMobilizerReactionOnParentAtOriginInGround(s);
```

Comment:
yes, per your question in the comments section, this is the right approach. The child mobilized body matches the mobilizer of the joint. Asking it for it's child and parent reaction loads is appropriate.